web/src/pages/apd/previous-activities/PreviousActivitiesReadOnly.js

```diff
@@ -20,9 +21,11 @@ const PreviousActivities = ({ previousActivitySummary, apdType }) => {
       <div dangerouslySetInnerHTML={{ __html: previousActivitySummary }} />
       <hr className="subsection-rule" />
       <h3>Actual Expenditures</h3>
-      {apdType === 'HITECH' && <ApdPreviousActivityTableHI isViewOnly />}
-      <ApdPreviousActivityTableMMIS isViewOnly />
-      <ApdPreviousActivityTableTotal />
+      {apdType === 'HITECH' && <HitechApdPreviousActivityTables isViewOnly />}
+      {apdType === 'HITECH' && <HitechPreviousActivityTotalsTable />}
+
+      {apdType === 'MMIS' && <MmisApdPreviousActivityTable isViewOnly />}
+      {apdType === 'MMIS' && <MmisPreviousActivityTotalsTable isViewOnly />}
```

Comment:
This totally works, so no need to change! Just wanted to offer another option for handling this kind of logic- I love a good mapping, so you could add an object like:
```
const activityTableMapping = {
  [APD_TYPE.HITECH]: <HitechApdPreviousActivityTables isViewOnly />,
  [APD_TYPE.MMIS]: <MmisApdPreviousActivityTable isViewOnly />
}
```
and then in the render you can reference the correct component by `activityTableMapping[apdType]`.
Ultimately it's just a nice way to be able to simplify number of lines in the render and also set the code up for easy expansion of adding more options to the mapping in the future.

Reply:
will make this change, thanks!